brave/app/scripts/controllers/infura.js

```diff
@@ -0,0 +1,19 @@
+const InfuraController = require('../../../../app/scripts/controllers/infura')
+
+module.exports = class BraveInfuraController extends InfuraController {
+
+  /*
+   * The indicator property represents the severity of any
+   * outages or downtime. "none" indicates an "ok" status
+   */
+  async checkInfuraNetworkStatus () {
+    const response = await fetch('https://hfq2h9152m63.statuspage.io/api/v2/status.json')
```

Comment:
what's this subdomain for? `hfq2h9152m63`

Reply:
@bbondy that is Infura's live status.io subdomain https://infura.statuspage.io/api#status

It is intended for public use, they just don't seem to host their own directly